utils/build-dists.py

```diff
@@ -194,7 +194,7 @@ def main() -> None:
         if m:
             version = m.group(1)
         else:
-            raise Exception(f"Invalid version {data}")
+            raise Exception(f"Invalid version: {data}")
```

Comment:
Did we get this exception? Or did the code just change the version to 0.0.0?

Reply:
I just changed the text to match another place we throw it.